tests/integration_tests/clouds.py

```diff
@@ -92,9 +91,12 @@ def from_os_image(cls):
 class IntegrationCloud(ABC):
     datasource: str
     cloud_instance: BaseCloud
+    instance_tag: str
+    pycloudlib_instance_cls = Type(BaseCloud)
```

Comment:
This fails for me (Python 3.10):
```
tests/integration_tests/conftest.py:16: in <module>
    from tests.integration_tests.clouds import (
tests/integration_tests/clouds.py:91: in <module>
    class IntegrationCloud(ABC):
tests/integration_tests/clouds.py:95: in IntegrationCloud
    pycloudlib_instance_cls = Type(BaseCloud)
/usr/lib/python3.10/typing.py:941: in __call__
    raise TypeError(f"Type {self._name} cannot be instantiated; "
E   TypeError: Type Type cannot be instantiated; use type() instead
```
Replacing with `type()` works.

Reply:
I had a typo/thinko
```suggestion
    pycloudlib_instance_cls: Type[BaseCloud]
```